js/auth-form.js

```diff
@@ -0,0 +1,72 @@
+function submitLoginForm() {
+  clearErrors();
+  var email = document.getElementById("email-input").value;
+  var password = document.getElementById("password-input").value;
+
+  var foundErrors = false;
+
+  if (email === "") {
+    addError("Email cannot be blank");
```

Comment:
it's bad to use representation in logic, u need to make a callback, and then, when u use submitLoginForm() If there is any errors it will return a callback as addError func.